kfs-cam/src/main/java/edu/arizona/kfs/module/cab/document/service/impl/PurApInfoServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package edu.arizona.kfs.module.cab.document.service.impl;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import edu.arizona.kfs.module.cab.CabConstants;
+import org.kuali.kfs.module.cab.businessobject.PurchasingAccountsPayableItemAsset;
+import org.kuali.kfs.module.cab.document.exception.PurApDocumentUnavailableException;
+import org.kuali.kfs.module.purap.PurapPropertyConstants;
+import org.kuali.kfs.module.purap.businessobject.CreditMemoItem;
+import org.kuali.kfs.module.purap.businessobject.PaymentRequestItem;
+import org.kuali.kfs.module.purap.businessobject.PurchaseOrderItem;
+import org.kuali.kfs.module.purap.document.VendorCreditMemoDocument;
```

Comment:
Please change the import to edu.arizona.kfs.module.purap.doucment.VendorCreditMemoDocument to gain access to the Payment Method Code.